apps/mobile/src/features/send-refactor/send-sheet.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSheetNavigatorContext } from '@/common/sheet-navigator/sheet-navigator-provider';
+import { FullHeightSheet } from '@/components/full-height-sheet/full-height-sheet';
+import { Send } from '@/features/send-refactor/send';
+
+import { useHaptics } from '@leather.io/ui/native';
+
+// TODO: Ideally this should live on an upper level. Send as a feature should be agnostic of the environment
```

Comment:
If you think we should do this, lets create a Linear task to capture it and ditch the comment. 

Otherwise we are throwing more `TODO`s on the pile

Reply:
True. Now that we have several major features in sheets, we need a better way of handling global sheets. Ditch this comment altogether. Tracking here: [LEA-2380: Improve global sheet handling in the mobile app](https://linear.app/leather-io/issue/LEA-2380/improve-global-sheet-handling-in-the-mobile-app)